### PR TITLE
del the DEFINE_ALIAS of sigmoid_cross_entropy_with_logits

### PR DESCRIPTION
**Pull request by chajchaj**
### PR types
Function optimization

### PR changes
APIs

### Describe
del the DEFINE_ALIAS of sigmoid_cross_entropy_with_logits

### CODE REVIEW

**Review by shippingwang — DISMISSED** [review]

**Review by willthefrog — APPROVED** [review]
LGTM, would recommend to consult maintainers of relevant models and establish upgrade path before merging this.

**Pull request merged**
